packages/dtslint/test/dt-header.test.ts

```diff
@@ -0,0 +1,168 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+
+import * as dtHeader from "../src/rules/dt-header";
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: "@typescript-eslint/parser",
+});
+
+ruleTester.run("no-const-enum", dtHeader, {
+  invalid: [
+    {
+      code: ``,
+      errors: [
+        {
+          column: 2,
+          data: {
+            expected: "/\\/\\/ Type definitions for (non-npm package )?/",
+          },
+          line: 1,
+          messageId: "parseError",
+        },
+      ],
+      filename: "types/abc/index.d.ts",
+    },
+    {
+      code: `
+      // ...
+      `,
+      errors: [
+        {
+          column: 2,
+          data: {
+            expected: "/\\/\\/ Type definitions for (non-npm package )?/",
+          },
+          line: 1,
+          messageId: "parseError",
+        },
+      ],
+      filename: "types/abc/index.d.ts",
+    },
+    {
+      code: `// Type definitions for dt-header 1.0
+// Project: https://github.com/bobby-headers/dt-header
+// Definitions by: Jane Doe <https://github.org/janedoe>
```

Comment:
in the absence of test names, maybe this would help call out the problem

```suggestion
// Definitions by: Jane Doe <https://github.ORG/janedoe>
```